src/main/java/com/nincraft/ninbot/components/ac/turnips/generator/BigSpikePriceGenerator.java

```diff
@@ -0,0 +1,38 @@
+package com.nincraft.ninbot.components.ac.turnips.generator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Pattern 1: decreasing middle, high spike, random low
+ */
+public class BigSpikePriceGenerator extends PatternPriceGenerator {
+
+    @Override
+    public List<Integer> generatePrices(int basePrice, Random random) {
+        RandomUtil randomUtil = new RandomUtil(random);
+        int index;
+        int peakStart = randomUtil.nextInt(3, 10);
+        double rate = randomUtil.nextDouble(0.85, 0.9);
+
+        List<Integer> prices = new ArrayList<>();
+
+        for (index = 0; index < peakStart; index++) {
+            prices.add((int) Math.ceil(rate * basePrice));
+            rate -= 0.03;
+            rate -= randomUtil.nextDouble(0, 0.02);
+        }
```

Comment:
This is duplicated in many spots. Could probably be pulled up to PatternPriceGenerator